syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -836,33 +791,28 @@ def unbind(tensor_shares, **kwargs):
         module.unbind = unbind
 
         def share_combine(tensors_shares):
-            results = {}
             workers = tensors_shares[0].keys()
 
-            for worker in workers:
-                tensors_share = []
-                for tensor_shares in tensors_shares:
-                    tensor_share = tensor_shares[worker]
-                    tensors_share.append(tensor_share)
-                results[worker] = tensors_share
-
-            return results
+            return {
+                worker: [tensor_shares[worker] for tensor_shares in tensors_shares]
+                for worker in workers
+            }
 
-        @overloaded.function
-        def stack(tensors_shares, **kwargs):
+        def make_worker_dict(op, tensors_shares, **kwargs):
```

Comment:
can we move it (together with share_combine) at a better place out of the `def torch` overloading? and maybe rename to `apply_on_shares`   on smthg like this?

Reply:
Not  sure as well - do yo think the multi_pointer.py could benefit from them?
If so we'll put hem in a generic place
if not, just put them in a staticmethod in the AST class but out of `def torch`